Cargo.toml

```diff
@@ -66,9 +66,10 @@ predicates = "0.9"
 difference = "2.0"
 
 [features]
-default = ["syntax-highlight", "sass"]
+default = ["syntax-highlight", "sass", "pagination-unstable"]
```

Comment:
Don't forget to remove this from default

Reply:
yup, I'll do this once everything is passing the tests :)